Dockerfile

```diff
@@ -12,8 +12,6 @@ RUN ./rclone version
 # Begin final image
 FROM alpine:latest
 
-LABEL org.opencontainers.image.source="https://github.com/rclone/rclone"
```

Comment:
Did you have a reason for reverting this? This was added recently in be53ae98f8ea54779cbfc85d1cf1ee7c04d1122b ?